### PR TITLE
Allow geonetwork logos to be configured using manager script

### PR DESCRIPTION
**Pull request by anguss00**
Deploy geonetwork config manager script with config file

### CODE REVIEW

**Comment by anguss00** [comment]
Bump

File: cookbooks/imos_webapps/files/default/geonetwork/geonetwork-config-manager.rb [hunk 1]
```diff
@@ -30,15 +31,17 @@
...
   uri = URI.parse(url)
-  http = Net::HTTP.new(uri.host, uri.port)
   req = Net::HTTP::Post.new(uri.request_uri)
   req.basic_auth(username, password)
   req.set_form_data(param_hash)
 
-  begin
-    res = http.request(req)
-  rescue
+  res = Net::HTTP::start(uri.hostname, uri.port) do |http|
```
Comment (pmbohm) [inline cookbooks/imos_webapps/files/default/geonetwork/geonetwork-config-manager.rb]: I dont know Ruby but can you have a `end` without `begin` ?

Comment (anguss00) [inline cookbooks/imos_webapps/files/default/geonetwork/geonetwork-config-manager.rb]: part of the do loop ``res = Net::HTTP::start(uri.hostname, uri.port) do |http|``

**Comment by anguss00** [comment]
fix up for 500 error @bhasin85 @pmbohm

**Comment by pmbohm** [comment]
It works. Nice one!

**Pull request merged**